packages/node/src/index.ts

```diff
@@ -27,20 +27,19 @@ export {
   withScope,
 } from '@sentry/core';
 
-export { NodeBackend, NodeOptions } from './backend';
+export { NodeOptions } from './backend';
 export { NodeClient } from './client';
 export { defaultIntegrations, init } from './sdk';
 export { SDK_NAME, SDK_VERSION } from './version';
 
 import { Integrations as CoreIntegrations } from '@sentry/core';
 import * as Handlers from './handlers';
 import * as NodeIntegrations from './integrations';
-import * as Parsers from './parsers';
 import * as Transports from './transports';
 
 const INTEGRATIONS = {
   ...CoreIntegrations,
   ...NodeIntegrations,
 };
 
-export { INTEGRATIONS as Integrations, Transports, Parsers, Handlers };
```

Comment:
Why do we hide `Parsers`? It could be used in some external frameworks integrations

Reply:
We have to think about this :/
Parser expose `SyncPromise` we not really want that, do we know who relies on the parsers?

Reply:
Yes, I for example rely on `parsers`. Creating a PR to expose it again.